### ISSUE TITLE
Split libretime package into smaller packages

### ISSUE THREAD

**Issue by jooola** [LibreTime/libretime-debian-packaging#16]
For maintenance purpose, more granular/advanced setup, simplicity, we could split the huge libretime package into multiple packages.

- libretime-playout (playout + liquidsoap)
- libretime-analyzer (analyzer)
- libretime-api (legacy + api  + worker (celery))
- libretime (depends on all the packages above)

Maybe we want something even more fine grained ?

It does require more packaging steps for the initial creation, but maintenance will probably be easier.

We could also create packages such as:
- libretime-apache-conf
- libretime-nginx-conf

To provide either a nginx installation or a Apache installation.

**Comment by paddatrapper** [comment]
Yeah, eventually I want a libretime meta-package with each component as a separate package under that. The issue at the moment is that distributing the deb files is much easier when they are all 1. Until we have a Debian apt repo, we can't really split it up (though I don't actually test the Ubuntu packages against Debian, so they may not work there anyway. In which case, that is mute and we can split it up immediately)
